File: Beginner/String Palindrome.java
import java.util.*;
class Palindrome {
    public static void main(String args[]) {
        // Your Code Here
		Scanner scanner=new Scanner(System.in);
		String line,check="";
    System.out.println("Enter the string to be checked :");
		line=scanner.nextLine();
		int n=line.length();
		for(int i=n-1;i>=0;i--){
			check+=line.charAt(i);
		}
		if(line.equals(check)){
			System.out.print("Palindrome");
		}
		else{
			System.out.print("Not Palindrome");
		}

    }
}

File: Beginner/StringPalindrome.c
#include <stdio.h>
#include <string.h>
 
int main()
{
  char a[100], b[100];
 
  printf("Enter a string to check if it is a palindrome\n");
  gets(a);
 
  strcpy(b, a);  
  strrev(b);  
 
  if (strcmp(a, b) == 0)  
    printf("The string is a palindrome.\n");
  else
    printf("The string isn't a palindrome.\n");
 
  return 0;
}
